coalib/bears/requirements/JuliaRequirement.py

```diff
@@ -29,3 +30,24 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'julia', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+        Creates the installation command for the instance of the class.
+
+        >>> JuliaRequirement("Lint").install_command()
+        'julia -e "Pkg.add(\"Lint\")'
```

Comment:
To get the expected output you need `JuliaRequirement("Lint", flag="-e").install_command()`